peachjam/views/home.py

```diff
@@ -24,15 +25,16 @@ def get(self, request, *args, **kwargs):
 
         authors = Author.objects.exclude(
             Q(genericdocument__isnull=True),
-            Q(judgment__isnull=True),
             Q(legalinstrument__isnull=True),
         )
 
+        courts = Court.objects.exclude(Q(judgment__isnull=True))
```

Comment:
I think this is being too smart, just include all the courts configured in the system.